### ISSUE TITLE
Unable to separate tile anymore

### ISSUE THREAD

**Issue by ahmadnazirafiq** [russtone/homebridge-mi-humidifier#49]
Recent update no longer allows me to separate the temperature & humidity tiles from the humidifier tiles

This makes it hard for us to discern whether the humidifier is on/off as the tiles’s color remains the same (unlike previous update where the tile turns white when on).

**Comment by russtone** [comment]
Please, provide more details on this.
Which updates do you mean as "recent" and "previous"?
What app do you use?
For me in Home.app everything works well

**Comment by mickgotwings** [comment]
So I was pretty sure it wasn't the plugin issue, but the Home app issue. However, I was curious enough to test this hypothesis.
I downgraded the plugin version gradually all the way down to 1.1.1 and the described problem persists.
I remember having this problem frustrate me long ago. Actually, since iOS 13.
From what I found out, this issue can be closed, as it has nothing to do with homebridge.
On second thought... @russtone, you might consider transmitting sensors to HomeKit as separate devices. This way they should not get grouped with the humidifier. It sounds counterintuitive, but it might be useful for many users (me included).
